doc/manual/substitutions.dist.py

```diff
@@ -7,6 +7,7 @@
 .. |DOMjudge| replace:: DOMjudge
 
 .. |baseurlteam| replace:: https://example.com/domjudge/team
+.. |problemseturlteam| replace:: https://example.com/domjudge/team/problems
```

Comment:
Do we need to mention the URL? It's in the menu, nobody should need to enter more than the base URL anyway.